node/coinstacks/solana/api/src/app.ts

```diff
@@ -1,17 +1,30 @@
 import express from 'express'
 import { join } from 'path'
 import swaggerUi from 'swagger-ui-express'
-import { middleware, Prometheus } from '@shapeshiftoss/common-api'
+import { ConnectionHandler, middleware, Prometheus, Registry, TransactionHandler } from '@shapeshiftoss/common-api'
 import { Logger } from '@shapeshiftoss/logger'
 import { RegisterRoutes } from './routes'
+import { Server } from 'ws'
+import { SolanaWebsocketClient } from './websocket'
+import { Helius } from 'helius-sdk'
+import { GeyserResultTransaction } from './models'
 
 const PORT = process.env.PORT ?? 3000
+const RPC_API_KEY = process.env.RPC_API_KEY
+const WEBSOCKET_URL = process.env.WEBSOCKET_URL
+const WEBSOCKET_API_KEY = process.env.WEBSOCKET_API_KEY
```

Comment:
need to add to the `sample.env`. would be ok to shorten `WEBSOCKET` -> `WS` as well imo

Reply:
Make sense, changed in [878a830](https://github.com/shapeshift/unchained/pull/1053/commits/878a830e579f9ac8933fc796bd7a059a25b62d7d)